sdk/cosmos/azure-cosmos/azure/cosmos/partition_key.py

```diff
@@ -38,14 +40,14 @@ class _Undefined(object):
 
 
 class PartitionKey(dict):
-    """ Key used to partition a container into logical partitions.
+    """Key used to partition a container into logical partitions.
 
     See https://docs.microsoft.com/azure/cosmos-db/partitioning-overview#choose-partitionkey
-    for more information on how to choose partition keys.
+    for information on how to choose partition keys.
 
     :ivar path: The path of the partition key
-    :ivar kind: What kind of partition key is being defined
-    :ivar version: The version of the partition key
+    :ivar kind: What kind of partition key is being defined (default: "Hash")
+    :ivar version: The version of the partition key (defatul: 2)
```

Comment:
Typo: "default"

Reply:
fixed in https://github.com/Azure/azure-sdk-for-python/pull/8607/commits/c14bb0d8adcb5e63ac1d6a2fab3819ccca50707e